CONTRIBUTING.md

```diff
@@ -21,6 +21,23 @@ automatically during the first build.
 
 Please also note that the project requires at least Java 8 in order to build successfully. But we don't think this is really an issue nowadays.
 
+### Project Structure Overview
+
+The project comprises five primary Gradle modules:
+
+- **bigbone:** This module contains the core code for interacting with the Mastodon REST API, including tests.
+- **bigbone-rx:** This module serves as a thin layer built on top of the "bigbone" module. It exposes methods available in the "bigbone" module as RxJava3 types, facilitating the use of this library within an RxJava context. Additionally, "bigbone-rx" enhances the functionality of "bigbone" by providing support for reactive streams.
```

Comment:
> It exposes methods available in the "bigbone" module as RxJava3 types, facilitating the use of this library within an RxJava context. **Additionally**, "bigbone-rx" enhances the functionality of "bigbone" by providing support for reactive streams.

This feels like a duplication. By exposing methods as RxJava3 types we’re _already_ saying that we provide support for reactive streams

```suggestion
- **bigbone-rx:** This module serves as a thin layer built on top of the `bigbone` module. It exposes methods available in the `bigbone` module as RxJava3 types, allowing the use of this library within an RxJava context.
```

Reply:
thanks for feedback. I did the changes, that you suggested 🙂